packages/core/admin/ee/server/routes/features-routes.js

```diff
@@ -43,4 +43,18 @@ module.exports = {
       },
     },
   ],
+  'review-workflows': [
+    {
+      method: 'GET',
+      path: '/review-workflows/workflows',
+      handler: 'workflows.find',
+      config: { auth: false },
+    },
+    {
+      method: 'GET',
+      path: '/review-workflows/workflows/:workflow_id',
```

Comment:
What do you think of putting just `:id`?
To be consistent with the rest of Strapi